src/content/blog/web-framework-performance-report.mdx

```diff
@@ -0,0 +1,132 @@
+
+All frameworks scored 50% or higher in this metric. However, it's the youngest frameworks (Astro, SvelteKit and Remix) that score the highest on this metric. All three scored over 75% on the assessment of this metric across all websites tested.
+
+<CWVLCPChart />
+
+[Largest Contentful Paint (LCP)](https://web.dev/lcp/) is the last of the three Core Web Vitals, and arguably the most important when it comes to perceived performance. It measures the point when the page's main content has likely loaded. An LCP of 2.5 seconds or less is required to pass Google's CWV Assessment. Anything slower is considered needing improvement and failing the assessment.
+
+LCP is the hardest of the three metrics to master. Only 52% of all websites tested pass this metric. Of our six frameworks tested, only Astro and SvelteKit beat this average. The rest come in below the average.
+
+# Coming Soon: Interaction to Next Paint (INP)
+
+**Interaction to Next Paint (INP)** is an experimental web vital that assesses overall website responsiveness, similar to First Input Delay (FID). Where the two metrics differ is that INP observes the latency of all interactions a user has made with the page, not just the first one. It also A low INP means the page was consistently able to respond quickly to all—or the vast majority—of user interactions.
+
+While INP is not an official core web vital today, the Chrome team has signaled [their hope](https://www.youtube.com/watch?v=Mizzbsvv8Os&t=378s) to replace FID with INP as a more holistic, accurate measure of responsiveness.
+
+So, how do our frameworks stack up against this new responsiveness metric?
+
+<CWVINPChart />
+
+Most noticible in the chart is that a good INP measurement is overall _much_ harder for our frameworks to achieve than First Input Delay (FID). While every framework tested saw an 80%+ pass rate of FID, no framework was able to see that same 80% pass rate on INP. Astro came closest, at 68.8% passing.
+
+It's worth noting that the average pass rate across all tracked websites is a suprisingly high 60.9%. While Astro and Wordpress look like the standout successes in the chart above, these sites are actually only performing modestly above the industry average. Why do many of the web frameworks tested struggle with this metric?
+
+One reason could be that Single Page App (SPA) architecture drives all navigation through JavaScript as a client-side action. This creates an opporunity for input delay that Multi-Page Apps (MPA) without client-side navigation don't have. In an MPA, navigating to a new page triggers a full page load from the server which isn't categorized as input delay. This could help explain why Astro and Wordpress (the two MPAs in the chart) perform significantly better on this metric than the rest of the frameworks tested (all SPAs).
```

Comment:
Typo: `opporunity` -> `opportunity`
Typo: `Wordpress` -> `WordPress`
```suggestion
One reason could be that Single Page App (SPA) architecture drives all navigation through JavaScript as a client-side action. This creates an opportunity for input delay that Multi-Page Apps (MPA) without client-side navigation don't have. In an MPA, navigating to a new page triggers a full page load from the server which isn't categorized as input delay. This could help explain why Astro and WordPress (the two MPAs in the chart) perform significantly better on this metric than the rest of the frameworks tested (all SPAs).
```